### PR TITLE
Release updated CifSubsurface and CifUnits schemas

### PR DESCRIPTION
**Pull request by Bordoloi123**

### CODE REVIEW

**Comment by diegoalexdiaz** [comment]
@Bordoloi123 please increment the version number on the "WIP" copy for these two schemas (i.e. CifUnits-->1.0.4 and CifSubsurface --> 1.0.7) as part of this PR.

**Comment by diegoalexdiaz** [comment]
BTW, you need to rerun "npm run updateSchemaInventory" after incrementing schema-version in order to keep the json db up to date.

**Review by jonathandecarlo — DISMISSED** [review]

File: Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifSubsurface.01.00.06.ecschema.xml [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+======================================================================================= -->
+<ECSchema schemaName="CifSubsurface" alias="cifssuf" version="01.00.06" description="iModel Connector schema containing aspect classes with properties from the Subsurface Utilities module as part of Civil Designer applications." xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="BisCore" version="01.00.14" alias="bis"/>
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="CifCommon" version="01.00.03" alias="cifcmn"/>
+    <ECSchemaReference name="CifUnits" version="01.00.01" alias="cifu"/>
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
```
Comment (ColinKerr) [inline Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifSubsurface.01.00.06.ecschema.xml:14]: Did you intend to switch this schema from Production to NotForProduction

File: Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifUnits.01.00.03.ecschema.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+======================================================================================= -->
+<ECSchema schemaName="CifUnits" alias="cifu" version="01.00.03" description="Unit definitions that are used across Civil Infrastructure Framework (CIF) iModel Connector schemas" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="Formats" version="01.00.00" alias="f"/>
+    <ECSchemaReference name="Units" version="01.00.05" alias="u"/>
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
```
Comment (ColinKerr) [inline Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifUnits.01.00.03.ecschema.xml:14]: Same here

File: Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifSubsurface.01.00.06.ecschema.xml (hunk 1 above)
Comment (Bordoloi123) [reply Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifSubsurface.01.00.06.ecschema.xml]: Changed to Production.

File: Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifUnits.01.00.03.ecschema.xml (hunk 2 above)
Comment (Bordoloi123) [reply Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifUnits.01.00.03.ecschema.xml]: Changed to Production

**Review by naveedkhan8067 — APPROVED** [review]

**Review by jonathandecarlo — APPROVED** [review]

**Pull request merged**